### ISSUE TITLE
I publish a Pytorch version, welcom to follow.

### ISSUE THREAD

**Issue by shoutOutYangJie** [znxlwm/UGATIT-pytorch#60]
Thank you. The code and training code can be found here. 
https://github.com/shoutOutYangJie/Morph-UGATIT

**Comment by markduon** [comment]
Dear sir,

I am a little confused with 2 CAM loss functions (5) and (6) in the paper. I don't know why one function uses BCE for generator, one uses MSE for discriminator? Can someone explain it? Many thanks.

**Comment by shoutOutYangJie** [comment]
> Dear sir,
> 
> I am a little confused with 2 CAM loss functions (5) and (6) in the paper. I don't know why one function uses BCE for generator, one uses MSE for discriminator? Can someone explain it? Many thanks.

The two CAM losses have a different effects.  The CAM in Generator helps to generate more realistic image. The CAM in D helps to distinct the real image and fake image.

**Comment by markduon** [comment]
Thank you for your response, sir!
I also do not know that what is the main function of auxiliary classifier in Generator and Discriminator? Are they the same in both?
